tests/layer_tests/tensorflow_lite_tests/test_tfl_Unary.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import itertools
+from functools import partial
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tflite_layer_test_class import TFLiteLayerTest
+
+np.random.seed(42)
+
+
+def make_positive_array(inputs_dict):
+    for input in inputs_dict.keys():
+        inputs_dict[input] = np.random.randint(1, 10, inputs_dict[input]).astype(np.float32)
+    return inputs_dict
+
+
+def make_boolean_array(inputs_dict):
+    for input in inputs_dict.keys():
+        inputs_dict[input] = np.random.randint(0, 1, inputs_dict[input]) > 1
+    return inputs_dict
+
+
+data_generators = {
+    'positive': make_positive_array,
+    'boolean': make_boolean_array,
+}
+
+test_ops = [
+    {'op_name': 'ABS', 'op_func': tf.math.abs},
```

Comment:
why don't we test it via TF layer test environment this case?

Reply:
we build operation coverage in the tensorflow_lite/

Reply:
Not quite clear why for this cases you don't use already implemented test `test_tf_UnaryOps.py` and you implemented infra to test TF Lite via these tests?

Reply:
We need this info on which operation types we are testing in terms of TFL. This infra helps with that. Also we don't need the test cases and marks (xfail, skip, etc) from tf.